homework/main.js

```diff
@@ -0,0 +1,8 @@
+'use strict';  
+   
+ document.addEventListener('DOMContentLoaded', function(event) {  
+     var nContent = new content();  
```

Comment:
**You need to learn to respect your work, even it's only a homework**.

To name things like this, and without any comment about the reason why you choose such special convention, is just show you are inexperienced and arbitrary. Although it is true that you are still a student, but please consider that most of your classmates have followed the good convention to complete their homework. **So please rename your constructors and instances with some meaningful and clear names**. If you don't have any idea, check my slides.